### ISSUE TITLE
Any updates on this? I was checking guix for AWS

### ISSUE THREAD

**Issue by ieugen** [ofosos/guix-packer#2]
Hi,

I read your post from 2018 and it got me to this repo.
Any updates on providing guix images for AWS or any other cloud?

**Comment by ofosos** [comment]
Not really. There was a low level of interest in having AWS images available for Guix.

People were especially suspicious of injecting configuration from the meta-data service into the running machine.

**Comment by ieugen** [comment]
Ok, thanks. 
Strange.
This is how cloud works. 
I don't see how you can trust the metadata service and trust the cloud provider. 
The metadata address is in a private network 169.x.x.x if I recall.

**Comment by ofosos** [comment]
It's a private address. It's essentially answered by the hypervisor (or parts thereof). The traffic should never leave the host. At least that's the way I think it still works. There's essentially no way you can not trust the hypervisor.

**Comment by rekado** [comment]
I'd still be interested in seeing this completed.
